packages/npm-dependencies-validation/src/logger.ts

```diff
@@ -1,9 +1,5 @@
 import { OutputChannel } from "./types";
 
-export function print(data: string, outputChannel?: OutputChannel): void {
-  if (outputChannel) {
+export function print(data: string, outputChannel: OutputChannel): void {
     outputChannel.appendLine(data);
-  } else {
```

Comment:
This logger/print is not very generic currently.
And also we don't actually need it right now, so it is simpler to remove it
instead of adding a test case for it...